modules/adot/variables.tf

```diff
@@ -48,3 +48,8 @@ variable "region" {
   description = "AWS Region"
   type        = string
 }
+
+variable "prometheus_metrics" {
```

Comment:
please, add a default value or type definition more detailed for having a simple example how this variable should be defined